### PR TITLE
Move Line Item Actions to solidus_legacy_promotions

### PR DESCRIPTION
**Pull request by mamhoff**
## Summary

This table belongs to the legacy promotion system and was, unfortunately, missed in the original extraction.

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [x] I have written a thorough PR description.
- [x] I have kept my commits small and atomic.
- [x] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

The following are not always needed:

- 📖 I have updated the README to account for my changes.
- 📑 I have documented new code [with YARD](https://www.rubydoc.info/gems/yard/file/docs/Tags.md).
- 🛣️ I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- ✅ I have added automated tests to cover my changes.
- 📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

File: legacy_promotions/db/migrate/20160101010001_solidus_one_four_promotions.rb [hunk 1]
```diff
@@ -122,5 +122,17 @@ def up
         t.index ["starts_at"], name: "index_spree_promotions_on_starts_at"
       end
     end
+
+    unless table_exists?(:spree_line_item_actions)
```
Comment (tvdeyen) [inline legacy_promotions/db/migrate/20160101010001_solidus_one_four_promotions.rb:126]: I think we need a new migration file now that we already shipped this gem 🙈

**Review by tvdeyen — COMMENTED** [review]
Ah, thanks! We cannot be sure that the migration has not already ran for folks. I think we should introduce a new migration file for this table

File: legacy_promotions/db/migrate/20160101010001_solidus_one_four_promotions.rb (hunk 1 above)
Comment (mamhoff) [reply legacy_promotions/db/migrate/20160101010001_solidus_one_four_promotions.rb:126]: I think we actually don't. This is just moved from the rollup migration for solidus one four in core to the one in solidus_legacy_promotions, and people who have installed Solidus have the table.

Comment (tvdeyen) [reply legacy_promotions/db/migrate/20160101010001_solidus_one_four_promotions.rb:126]: Right

**Review by tvdeyen — APPROVED** [review]

**Review by kennyadsl — APPROVED** [review]
👍 going in the next patch level release?

**Pull request merged**